mobilecoind/src/payments.rs

```diff
@@ -826,12 +825,21 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         let idx = self.submit_node_offset.fetch_add(1, Ordering::SeqCst);
         let responder_id = &responder_ids[idx % responder_ids.len()];
 
+        // The rationale for these retries is:
+        // * Quite often, the attested connetion was closed and we need to retry once to
+        //   re-attest, and this attestation is successful. So that takes 50 ms.
+        // * After that, we back off to 500 ms, because there are rate limits of about
+        //   100 / min in place in production.
+        let retry_iterator = (&[50, 500, 500, 750, 1000])
+            .iter()
+            .map(Duration::from_millis);
```

Comment:
I am not sure what the most appropriate strategy, if you want to go back to fibonacci backoff (50 ms, 5 tries) I am also fine with that.

I would like to merge something so that I can bump deqs repo to get working mobilecoind and not have to run my demos with patches, that's all

Reply:
it also feels wierd that a "submit tx" request can hang for 3.8 seconds, but maybe that's fine

Reply:
> Like, repeated attestation failures should go wild and get throttled, right?

@jgreat what is the downside of this if any?

Reply:
To add to this, the sitsuation were we get a permission denied error and what we do about it my vary.  

If its a watcher connection that is monitoring the consensus nodes, then a back off and retry until recovery or intervention might make sense.  

Submitting a transaction may want a fast retry/fail pattern and ask the user to try again later.

Reply:
thanks for your thoughts, let's make an issue to follow up